uk.co.piggz.amazfish.json

```diff
@@ -0,0 +1,120 @@
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--system-talk-name=org.bluez",
+    "--talk-name=org.mpris.MediaPlayer2.*",
+    "--talk-name=io.github.rinigus.PureMaps",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "nemo-qml-plugin-dbus",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "cleanup": [
+        "/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/nemo-qml-plugin-dbus/archive/refs/tags/2.1.33.tar.gz",
+          "sha256": "08d1bdc5eac5eba2197f2c1e04a7e851437724331215059db1d68828dbf3b0a8"
+        },
+        {
+          "type": "patch",
+          "path": "nemodbus-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "qtmpris",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "cleanup": [
+        "/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/qtmpris/archive/refs/tags/1.0.6.tar.gz",
+          "sha256": "384eac76ec7d29d1de6fa93c45007ccd506d6cc0e4c69e19d4250e16cc0f34a2"
+        },
+        {
+          "type": "patch",
+          "path": "qtmpris-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "libkdb3",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
+      "cleanup": [
+        "/include",
+        "/mkspecs"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/libraries/kdb/-/archive/tags/v3.2.0/kdb-v3.2.0.tar.gz",
+          "sha256": "3a5c660f0080c6971ac1046be84a0accd86a5296e9df4c585b1b91361b5eed20"
+        },
+        {
+          "type": "patch",
+          "path": "6.patch"
+        },
+        {
+          "type": "patch",
+          "path": "upstream_Fix-build-with-newer-Qt.patch"
+        },
+        {
+          "type": "patch",
+          "path": "libkdb3-features.patch"
+        }
+      ]
+    },
+    {
+      "name": "harbour-amazfish",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "FLAVOR=qtcontrols"
+      ],
+      "build-options": {
+        "env": [
+          "QMAKEPATH=/app"
+        ]
+      },
+      "post-install": [
+        "install -Dm755 run-amazfish.sh /app/bin/run-amazfish.sh",
+        "install -Dm644 ui/icons/128x128/harbour-amazfish-ui.png /app/share/icons/hicolor/128x128/apps/harbour-amazfish-ui.png",
+        "install -Dm644 ui/harbour-amazfish-ui.appdata.xml /app/share/appdata/harbour-amazfish-ui.appdata.xml",
+        "desktop-file-edit --set-key=Exec --set-value='run-amazfish.sh' /app/share/applications/harbour-amazfish-ui.desktop"
+      ],
+
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/piggz/harbour-amazfish.git",
+          "branch": "2.2.5"
```

Comment:
Branch is not allowed.

Reply:
I know, we are waiting for upstream obviously. This is just to have idea how it will look like in the end.